### PR TITLE
Init announcement

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/announcing-init/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: "Announcing Init, what's next?"
+description: 
+date: 2024-02-19
+cover: /images/blog/init-announcement.png
+timeToRead: 4
+author: laura-du-ry
+category: open-source
+---
+
+We are very excited to finally announce Init.
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-init/+page.markdoc:12]:
```suggestion
We are very excited to announce Init. 
```

File: src/routes/blog/post/announcing-init/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+date: 2024-02-19
+cover: /images/blog/init-announcement.png
+timeToRead: 4
+author: laura-du-ry
+category: open-source
+---
+
+We are very excited to finally announce Init. 
+
+A new way to experience Appwrite releases, starting with Appwrite 1.5.
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-init/+page.markdoc:14]: Not just releases. The focus is around release, updates, news, content. Everything that is new in Appwrite. Init is the start of something new.

File: src/routes/blog/post/announcing-init/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+### Celebrating innovation and technology
+
+You bring innovation and technology to life that impacts daily lives. Innovation starts on the brink of something new, and Init is a time to start new ideas, or, even better, initialize new ideas. Whether it be a new project, a contribution, a tutorial, a function, or even a new business, you’re initializing what is next.
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-init/+page.markdoc:18]: Life and lives twice sounds off.

File: src/routes/blog/post/announcing-init/+page.markdoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+### Get your ticket and join the fun
+
+Init gives life to new ideas and celebrates them, and we’re excited to see it inspire your next creation. 
+
+Init takes place in the week of 26 February to 01 March 2024. 
+
+"Don't miss out! Be sure to register and claim your ticket at [appwrite.io/init/ticket](appwrite.io/init/ticket)."
+
+
+**"Init_ what’s next?"**
```
Comment (eldadfux) [inline src/routes/blog/post/announcing-init/+page.markdoc:43]: I like the slogan `Init is the start of something new better` Let's be more consistent with whatever message across all channels.

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: src/routes/blog/post/announcing-init/+page.markdoc (hunk 2 above)
Comment (LauraDuRy) [reply src/routes/blog/post/announcing-init/+page.markdoc:14]: Updated to: A new way to experience everything new with Appwrite, starting with Appwrite 1.5.

File: src/routes/blog/post/announcing-init/+page.markdoc (hunk 4 above)
Comment (LauraDuRy) [reply src/routes/blog/post/announcing-init/+page.markdoc:43]: updated

File: src/routes/blog/post/announcing-init/+page.markdoc (hunk 3 above)
Comment (LauraDuRy) [reply src/routes/blog/post/announcing-init/+page.markdoc:18]: adjusted

**Pull request merged**